pkgs/by-name/tu/tuatara/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zig_0_13,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tuatara";
+  version = "1631040452-unstable-2025-04-29";
+
+  src = fetchFromGitHub {
+    owner = "q60";
+    repo = "tuatara";
+    rev = "bc093e5fe1cb8dec667806f1b41c8e4e913368e8";
+    hash = "sha256-GLOb2vqDlcCQ3bPXC50t1j+DJFhl8JK117t7uRLrBbk=";
+  };
+
+  nativeBuildInputs = [ zig_0_13 ];
+
+  strictDeps = true;
+
+  buildPhase = ''
+    runHook preBuild
+
+    mkdir -p zig-out
+    zig build -Doptimize=ReleaseFast
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 -t $out/bin zig-out/bin/${finalAttrs.pname}
+
+    runHook postInstall
+  '';
```

```suggestion
  nativeBuildInputs = [ zig_0_13.hook ];
```
And that's it, it already sets up build and install